rllib/algorithms/algorithm_config.py

```diff
@@ -1305,3 +1599,123 @@ def experimental(
             self._disable_execution_plan_api = _disable_execution_plan_api
 
         return self
+
+    def __setattr__(self, key, value):
```

Comment:
These methods help make AlgorithmConfigs still appear as dicts (in case some of RLlib's Algorithms or users in their custom algos still use configs like dicts).
These should be removed in the future.